addons/stock/models/stock_location.py

```diff
@@ -95,6 +97,11 @@ def _compute_next_inventory_date(self):
             else:
                 location.next_inventory_date = False
 
+    @api.depends('location_id.warehouse_id', 'warehouse_view_ids')
```

Comment:
`location_id.warehouse_id` is not a stored field, Will it trigger compute method?